swift/apple/FirezoneKit/Sources/FirezoneKit/Stores/Store.swift

```diff
@@ -323,6 +324,12 @@ public final class Store: ObservableObject {
             if !Task.isCancelled {
               do {
                 self.resourceList = try await self.ipcClient().fetchResources()
+              } catch let error as NSError {
+                if error.domain == "NEVPNErrorDomain" && error.code == 1 {
```

Comment:
Is `1` a documented constant?

Reply:
Updated with a link to https://developer.apple.com/documentation/networkextension/nevpnerror-swift.struct/code